BT-Tracking/Interface/Data List/DataListVC.swift

```diff
@@ -128,14 +127,14 @@ final class DataListVC: UIViewController, UITableViewDelegate {
             )
         }))
         controller.preferredAction = controller.actions.first
-        present(controller, animated: true, completion: nil)
+        present(controller, animated: true)
     }
 
 }
 
 private extension DataListVC {
 
-    func sendReport() {
+    private func sendReport() {
```

Comment:
Tady je private extension, to private je tu pridane zbytecne

Reply:
Toho jsem si nevsiml. Opravim.